src/components/MultipleAvatars.js

```diff
@@ -1,7 +1,9 @@
 import React, {memo} from 'react';
 import PropTypes from 'prop-types';
 import {Image, Text, View} from 'react-native';
+import _ from 'lodash';
```

Comment:
We usually do `import lodashGet from 'lodash.get';` so it would be good to keep that pattern for consistency.